src/Datadog.Trace.AspNet/HttpApplicationStartup.cs

```diff
@@ -1,35 +0,0 @@
-using System.Web;
-using Datadog.Trace.AspNet;
-
-[assembly: PreApplicationStartMethod(typeof(HttpApplicationStartup), "Register")]
```

Comment:
I thought this was needed to load the TracingHttpModule, but looks like it works without it

Reply:
It's deprecated code from when we used to distribute it as a nuget package.

Reply:
We now add it to `applicationHost.config` during installation. Remember the bug when we left out the `v` in `runtimeVersionv4.0`? 😅 

https://github.com/DataDog/dd-trace-dotnet/blob/4d7cbd6710bf6a66bbae4c7005b153cfd2f97f90/src/WindowsInstaller/install.cmd#L6